src/vue/backend/src/views/guided-setup/Listings.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+    <GuidedSetupContentCard
+        :cardTitle="cardTitle"
+        :steps="guidedSetupSteps"
+        :relatedLinks="links"
+        @back-step="goBackStep"
+        @skip-step="goSkipStep"
+        @continue="goContinue">
+        <template v-slot:description>
+            <p><strong>This is optional.</strong> A sentence or two about why you should install IMPress Listings to your WordPress site. Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+        </template>
+        <template v-slot:controls>
+            <p>Activating IMPress Listings will:</p>
+            <idx-list customClass="list-featured">
+                <idx-list-item>Feature 1</idx-list-item>
+                <idx-list-item>Feature 2</idx-list-item>
+                <idx-list-item>Feature 3</idx-list-item>
+                <idx-list-item>Feature 4</idx-list-item>
+            </idx-list>
+            <idx-block className="form-content">
+                <idx-form-group>
+                    <idx-block className="form-content__toggle">
+                        {{ activateLabel }}
+                        <idx-toggle-slider
+                            uncheckedState="No"
+                            checkedState="Yes"
+                            @toggle="listingsSettingsStateChange({ key: 'enabled', value: !enabled })"
+                            :active="enabled"
+                            :label="activateLabel"
+                        ></idx-toggle-slider>
+                    </idx-block>
+                </idx-form-group>
+            </idx-block>
+        </template>
+    </GuidedSetupContentCard>
+</template>
+
+<script>
+import { mapState, mapActions } from 'vuex'
+import GuidedSetupContentCard from '@/templates/GuidedSetupContentCard.vue'
+export default {
+    components: {
+        GuidedSetupContentCard
+    },
+    data () {
+        return {
+            cardTitle: 'Activate IMPress Listings',
+            activateLabel: 'Activate',
```

Comment:
@ralls Are we wanting to update the verbiage to "Enable" Across the board?

Reply:
Yes but we can either do that now or when we decouple state from form data